pkg/resources/testdata/TestAcc_ExternalVolume/azure/complete/test.tf

```diff
@@ -0,0 +1,14 @@
+resource "snowflake_external_volume" "complete" {
+  name         = var.name
+  comment      = var.comment
+  allow_writes = var.allow_writes
+  dynamic "storage_location" {
+    for_each = var.storage_location
+    content {
+      storage_location_name = storage_location.value["storage_location_name"]
```

Comment:
I think the name of the directory should be `azure/no-encryption`. Similarly, this applies to other providers.

Reply:
Ended up deleting this in https://github.com/Snowflake-Labs/terraform-provider-snowflake/pull/3106/commits/68c104d594d26823233d5aeef3767c359004d37f as I consolidated many test directories